scripts/install_deepchem_conda.sh

```diff
@@ -33,7 +33,8 @@ conda install -y -q -c deepchem -c rdkit -c conda-forge -c omnia \
     py-xgboost \
     rdkit \
     simdna \
+    pymatgen \
     pytest \
     pytest-cov \
     flaky
-yes | pip install -U tensorflow tensorflow-probability
+yes | pip install -U matminer tensorflow tensorflow-probability
```

Comment:
Can you also add this to `scripts/install_deepchem_conda.ps1`?

Reply:
Will do!